docs/api/createAsyncThunk.mdx

```diff
@@ -89,6 +89,7 @@ An object with the following optional fields:
 
 - `condition`: a callback that can be used to skip execution of the payload creator and all action dispatches, if desired. See [Canceling Before Execution](#canceling-before-execution) for a complete description.
 - `dispatchConditionRejection`: if `condition()` returns `false`, the default behavior is that no actions will be dispatched at all. If you still want a "rejected" action to be dispatched when the thunk was canceled, set this flag to `true`.
+- `idGenerator`: a function to use when generating the `requestId` for the request sequence. Defaults to use [nanoid](https://www.npmjs.com/package/nanoid).
```

Comment:
:hand: Let's point this to `./otherExports.mdx#nanoid` instead, and maybe also tweak the wording of the `nanoid` paragraph to say it's the "default" instead of "automatically used"

Reply:
Both points now addressed

Reply:
I realised I made a sentence for the `nanoid` description incorrect when I adjusted it, but tweaked it to make sense again this time.